ambry-api/src/main/java/com/github/ambry/config/ClusterMapConfig.java

```diff
@@ -23,6 +23,8 @@
 public class ClusterMapConfig {
 
   public static final String CLUSTERMAP_CLUSTER_NAME = "clustermap.cluster.name";
+  public static final String CLUSTERMAP_AGGREGATED_VIEW_CLUSTER_NAME = "clustermap.aggregated.view.cluster.name";
+  public static final String CLUSTERMAP_USE_AGGREGATED_VIEW = "clustermap.use.aggregated.view";
```

Comment:
Can we switch to use the aggregated view only?  Is there reason why we want to keep the old way?

Reply:
Sure. I was thinking that we could have this config for now as an option to rollback in case we see any issues. We can probably remove it eventually?